### PR TITLE
Grammar

### PR DESCRIPTION
**Pull request by jsoref**
This isn't a complete audit. I've tried to walk through the document using Grammarly. The descriptions are my attempt at summarizing what Grammarly was saying.

### CODE REVIEW

File: draft-ietf-acme-acme.md [hunk 1]
```diff
@@ -120,7 +120,7 @@ means that under typical circumstances, all steps in the request, verification,
 and issuance process can be represented and performed by Internet protocols with
 no out-of-band human intervention.
 
-At time of writing, when deploying an HTTPS server, an operator typically gets a
+When this document was drafted, when deploying an HTTPS server, an operator typically gets a
```
Comment (jsha) [inline draft-ietf-acme-acme.md]: Let's change this to "Prior to ACME, "

File: draft-ietf-acme-acme.md [hunk 2]
```diff
@@ -2455,7 +2455,7 @@ document ]]
 ### Identifier Types {#iana-identifier}
 
 This registry lists the types of identifiers that ACME clients may request
-authorization to issue in certificates.
+authorization to issue certificates.
```
Comment (jsha) [inline draft-ietf-acme-acme.md]: This diff is not correct. I'd suggest reverting for now.

File: draft-ietf-acme-acme.md [hunk 3]
```diff
@@ -2547,12 +2547,12 @@ connections to the ACME server and the validation channel outbound HTTP or DNS
...
 
 Broadly speaking, ACME aims to be secure against active and passive attackers on
-any individual channel.  Some vulnerabilities arise (noted below), when an
+any individual channel.  Some vulnerabilities arise (noted below) when an
 attacker can exploit both the ACME channel and one of the others.
 
 On the ACME channel, in addition to network-layer attackers, we also need to
-account for application-layer man in the middle attacks, and for abusive use of
-the protocol itself.  Protection against application-layer MitM addresses
+account for man-in-the-middle attacks at the application-layer, and for abusive
```
Comment (jsha) [inline draft-ietf-acme-acme.md]: Replace application-layer with application layer.

File: draft-ietf-acme-acme.md [hunk 4]
```diff
@@ -2648,15 +2648,15 @@ An active attacker on the validation channel can subvert the ACME process, by
...
+associated with the domain, this attack can be prevented by requiring the client
 to prove control of the corresponding private key.
 
 ## Denial-of-Service Considerations
 
 As a protocol run over HTTPS, standard considerations for TCP-based and
 HTTP-based DoS mitigation also apply to ACME.
 
-At the application layer, ACME requires the server to perform a few potentially
+At the application-layer, ACME requires the server to perform a few potentially
```
Comment (jsha) [inline draft-ietf-acme-acme.md]: Undo this change.

File: draft-ietf-acme-acme.md (hunk 2 above)
Comment (jsoref) [inline draft-ietf-acme-acme.md]: Can you explain what this is trying to say?

File: draft-ietf-acme-acme.md (hunk 3 above)
Comment (jsoref) [inline draft-ietf-acme-acme.md]: There are instances of both network-layer and application-layer, do you want both to be unhyphenated?

File: draft-ietf-acme-acme.md (hunk 2 above)
Comment (jsoref) [inline draft-ietf-acme-acme.md]: This registry lists the types of identifiers in certificates that ACME clients may request authorization to issue.

File: draft-ietf-acme-acme.md (hunk 3 above)
Comment (jsha) [inline draft-ietf-acme-acme.md]: Yep, both should be unhyphenated. I think Grammerly is probably just unfamiliar with "network" and "application" as adjectives.

File: draft-ietf-acme-acme.md (hunk 2 above)
Comment (jsha) [inline draft-ietf-acme-acme.md]: Yep, your revision sounds great.

File: draft-ietf-acme-acme.md (hunk 3 above)
Comment (jsoref) [inline draft-ietf-acme-acme.md]: I think it might have been happier if the document was consistent. I'm not sure.

**Comment by jsoref** [comment]
I've updated this PR to address the review.

**Comment by jsha** [comment]
Looks good, will merge.

For future reference, here's how I would prefer pull requests:

- Provide a branch with one or more commits. Use a meaningful description and summary for the PR.
- Provide updates as new commits, not rebases.
- When merging, I will squash all commits into one, using the PR description and summary as the commit message.

**Pull request merged**